src/unix/bsd/mod.rs

```diff
@@ -148,8 +148,7 @@ cfg_if! {
     if #[cfg(any(target_os = "macos", target_os = "ios"))] {
         mod apple;
         pub use self::apple::*;
-    } else if #[cfg(any(target_os = "openbsd", target_os = "netbsd",
-                        target_os = "dragonfly"))] {
+    } else if #[cfg(any(target_os = "openbsd", target_os = "netbsd"))] {
```

Comment:
hi @mneumann 

I have a conflicted diff for this file in #39. Could you adjust your PR ? Just replacing "dragonfly" by "bitrig" (in the "openbsd" line)